data/pc/1.20.5/entities.json

```diff
@@ -0,0 +1,1302 @@
+[
```

Comment:
ok for now, but there is additional data to extract here with the extractor in tools/js/extractMcpcEntityMetadata for extracting metadata keys (used in mineflayer)